persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/StructuralVariantMyBatisRepositoryTest.java

```diff
@@ -403,4 +408,60 @@ public void fetchStructuralVariantsMultiStudyByGeneQueriesWithSampleIdentifiersU
         Assert.assertArrayEquals(new String[] {"KIAA1549-BRAF.K16B10.COSF509", "NCOA4-RET.N7R1"}, resultTcgaVariants.toArray());
     }
     
+    @Test
+    public void fetchStructuralVariantsWithSingleStructuralVariantIdentifier() throws Exception {
+
+        List<String> molecularProfileIds = new ArrayList<>();
+        List<Integer> entrezGeneIds = new ArrayList<>();
+        List<String> sampleIds = new ArrayList<>();
+
+        molecularProfileIds.add("study_tcga_pub_sv");
+
+        List<StructuralVariantIdentifier> singleStructVarIdentifier = new ArrayList<>();
+        singleStructVarIdentifier.add(new StructuralVariantIdentifier(27436, 238));
+        
+        List<StructuralVariant> result = 
+                structuralVariantMyBatisRepository.fetchStructuralVariants(molecularProfileIds,
+                    sampleIds, entrezGeneIds, singleStructVarIdentifier
+                );
+        
+        Assert.assertEquals(1,  result.size());
+        StructuralVariant structuralVariantFirstResult = result.get(0);
+        Assert.assertEquals((Integer) 27436, structuralVariantFirstResult.getSite1EntrezGeneId());
+        Assert.assertEquals((Integer) 238, structuralVariantFirstResult.getSite2EntrezGeneId());
+        Assert.assertEquals("Fusion", structuralVariantFirstResult.getEventInfo());
+    }
+    
+    @Test
+    public void fetchStructuralVariantsWithMultipleStructuralVariantIdentifiers() throws Exception {
+        List<String> molecularProfileIds = new ArrayList<>();
+        List<Integer> entrezGeneIds = new ArrayList<>();
+        List<String> sampleIds = new ArrayList<>();
+
+        molecularProfileIds.add("study_tcga_pub_sv");
+
+        List<StructuralVariantIdentifier> multipleStructVarIdentifiers = new ArrayList<>();
+        multipleStructVarIdentifiers.add(new StructuralVariantIdentifier(27436, 238));
+        multipleStructVarIdentifiers.add(new StructuralVariantIdentifier(57670, 673));
+        
+        List<StructuralVariant> result = 
+                structuralVariantMyBatisRepository.fetchStructuralVariants(molecularProfileIds,
+                    sampleIds, entrezGeneIds, multipleStructVarIdentifiers
+                );
+        
+        Assert.assertEquals(3,  result.size());
+
+        List<StructuralVariant> structuralVariantFirstResult = result.stream()
+            .filter(sv -> Objects.equals(sv.getSite1EntrezGeneId(), 27436) 
+                && Objects.equals(sv.getSite2EntrezGeneId(), 238))
+            .collect(Collectors.toList());;
+        Assert.assertEquals(1,  structuralVariantFirstResult.size());
+
+        List<StructuralVariant> structuralVariantSecondResults = result.stream()
+            .filter(sv -> Objects.equals(sv.getSite1EntrezGeneId(), 57670) 
+                && Objects.equals(sv.getSite2EntrezGeneId(), 673))
+            .collect(Collectors.toList());
+        Assert.assertEquals(2, structuralVariantSecondResults.size());
+    }
+    
```

Comment:
Please add test cases for _any gene_ and _null gene_.

Reply:
Added in [4c93d75](https://github.com/cBioPortal/cbioportal/commit/4c93d75879595db2e4add0ff8452a5647b540756)